src/Card.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components'
+import Box from './Box'
+import PropTypes from 'prop-types'
+
+function boxShadow(props) {
+
+  const boxShadows = {
+    'sm': {
+      'box-shadow': props.theme.boxShadows[0]
+    },
+    'md': {
+      'box-shadow': props.theme.boxShadows.slice(0,2).join(', ')
+    },
+    'lg': {
+      'box-shadow': props.theme.boxShadows.slice(0,3).join(', ')
+    },
+    'xl': {
+      'box-shadow': props.theme.boxShadows.join(', ')
+    }
+  }
```

Comment:
RE: the box-shadow sizes being concatenated - Oh I see, so based on your  I guess it's for design, to make the larger shadows softer. That makes sense.

Another thought: I'd want to see if there's any performance impact on from this when rendering a lot of cards at once compared to not layering multiple shadows per box, especially on mobile. Might not be a concern, but it could have an impact...

Reply:
There would definitely be some performance issues *if* we transitioned/animated a Card or changed box-shadow, but I suspect static box-shadows should be fine